docs/spec/spec.md

```diff
@@ -1652,14 +1655,22 @@ The `graphiql` field is used to provide the GraphiQL client configuration to ena
 @graphql:ServiceConfig {
     graphiql: {
         enabled: true,
-        path: "/ballerina/graphiql"
+        path: "/ballerina/graphiql",
+        printUrl: false
     }
 }
 service on new graphql:Listener(9090) {
     // ...
 }
 ```
->**Note:** The field enabled accepts a `boolean` that denotes whether the client is enabled or not. By default, it has been set to `false`. The optional field `path` accepts a valid `string` for the GraphiQL service. If the path is not given in the configuration, `/graphiql` is set as the default path.
+##### 7.1.5.1 Field Enabled
```

Comment:
Check the other places

```suggestion
##### 7.1.5.1 Field `enabled`
```

Reply:
I think the convention we followed in the other places is to use something like this:

```suggestion
##### 7.1.5.1 The `enabled` Field
```